From c9e108c75a27a3bf3a03d63b05596bf6be6f95e4 Mon Sep 17 00:00:00 2001
From: Mandy Lavi <mandy.lavi@freescale.com>
Date: Wed, 29 May 2013 14:12:37 +0300
Subject: [PATCH 202/987] fmd: changed field name from portParams to
 port_params

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Signed-off-by: Mandy Lavi <mandy.lavi@freescale.com>
Change-Id: I5efcdbe646486c9b1b0863a714fc49bc7d7ccb3f
Reviewed-on: http://git.am.freescale.net:8181/2754
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Chereji Marian-Cornel-R27762 <marian.chereji@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 .../ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.c | 8 ++++----
 .../ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.c b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.c
index a8e9f0b..7e29867 100644
--- a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.c
+++ b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.c
@@ -1129,8 +1129,8 @@ void compat_copy_fm_vsp_params(
         param->p_fm = compat_pcd_id2ptr(compat_param->p_fm);
         memcpy(&param->ext_buf_pools, &compat_param->ext_buf_pools, sizeof(ioc_fm_ext_pools));
         param->liodn_offset = compat_param->liodn_offset;
-        param->portParams.port_id = compat_param->portParams.port_id;
-        param->portParams.port_type = compat_param->portParams.port_type;
+        param->port_params.port_id = compat_param->port_params.port_id;
+        param->port_params.port_type = compat_param->port_params.port_type;
         param->relative_profile_id = compat_param->relative_profile_id;
         param->id = compat_pcd_id2ptr(compat_param->id);
     }
@@ -1139,8 +1139,8 @@ void compat_copy_fm_vsp_params(
         compat_param->p_fm = compat_pcd_ptr2id(param->p_fm);
         memcpy(&compat_param->ext_buf_pools, &param->ext_buf_pools, sizeof(ioc_fm_ext_pools));
         compat_param->liodn_offset = param->liodn_offset;
-        compat_param->portParams.port_id = param->portParams.port_id;
-        compat_param->portParams.port_type = param->portParams.port_type;
+        compat_param->port_params.port_id = param->port_params.port_id;
+        compat_param->port_params.port_type = param->port_params.port_type;
         compat_param->relative_profile_id = param->relative_profile_id;
         compat_param->id = compat_add_ptr2id(param->id, FM_MAP_TYPE_PCD_NODE);
     }
diff --git a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
index 3078003..288777a 100644
--- a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
+++ b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_ioctls_fm_compat.h
@@ -489,7 +489,7 @@ typedef struct ioc_compat_fm_vsp_params_t {
     struct {
         ioc_fm_port_type port_type;           /**< Port type */
         uint8_t          port_id;             /**< Port Id - relative to type */
-    } portParams;
+    } port_params;
     uint8_t             relative_profile_id;  /**< VSP Id - relative to VSP's range
                                                    defined in relevant FM object */
     compat_uptr_t       id;                 /**< return value */
-- 
1.9.1

